libs/wire-api/src/Wire/API/EnterpriseLogin.hs

```diff
@@ -100,7 +100,7 @@ domainRedirectSchemaV8 =
 backendConfigSchemaV8 :: ObjectSchema SwaggerDoc (HttpsUrl, Maybe HttpsUrl)
 backendConfigSchemaV8 =
   (,)
-    <$> fst .= backendUrlSchema
+    <$> fst .= field "backend_url" schema
     -- API versions <= V9 ignore the WebApp URL
```

Comment:
is this comment still correct?  (was it ever?  shouldn't it be `<9`?)

Reply:
what is the point of this change?

Reply:
> is this comment still correct? (was it ever? shouldn't it be `<9`?)

Thanks for spotting this! :+1:  You're right it should be `<=V8` or `<V9` (which are equal constraints.)

The change itself is just inlining a public function which is only used here. There's no second call site, thus we don't need add this indirection.